apps/condo/domains/property/components/AddressSuggestionsSearchInput.tsx

```diff
@@ -1,13 +1,15 @@
 import { grey } from '@ant-design/colors'
 import { useAddressApi } from '@condo/domains/common/components/AddressApi'
 import { BaseSearchInput } from '@condo/domains/common/components/BaseSearchInput'
+import { renderHighlightedPart } from '@condo/domains/common/components/Table/Renders'
+import { TextHighlighter } from '@condo/domains/common/components/TextHighlighter'
+import { validHouseTypes } from '@condo/domains/property/constants/property'
 import styled from '@emotion/styled'
 import { useIntl } from '@open-condo/next/intl'
 import { notification, Select, SelectProps } from 'antd'
 import get from 'lodash/get'
 import React, { Dispatch, SetStateAction, useCallback, useEffect, useState } from 'react'
```

Comment:
we use to sort imports

Reply:
Done

Reply:
like this

```
import React, { Dispatch, SetStateAction, useCallback, useEffect, useState } from 'react'

import { notification, Select, SelectProps } from 'antd'
import get from 'lodash/get'
import styled from '@emotion/styled'

import { useIntl } from '@open-condo/next/intl'

import { useAddressApi } from '@condo/domains/common/components/AddressApi'
import { BaseSearchInput } from '@condo/domains/common/components/BaseSearchInput'
import { renderHighlightedPart } from '@condo/domains/common/components/Table/Renders'
import { TextHighlighter } from '@condo/domains/common/components/TextHighlighter'

import { validHouseTypes } from '@condo/domains/property/constants/property'
import { TSelectedAddressSuggestion } from '@condo/domains/property/components/BasePropertyForm/types'
```

Reply:
Imports are sorted alphabetically. This type of sorting may be done automatically. Separating into groups is subjective.

Reply:
We used to sort imports like this:
1. Third-parties 
2. Packages
3. Domains
4. Relatives

Seems like not obvious :(

Reply:
Will be done with DOMA-5161